src/uct/api/uct.h

```diff
@@ -618,6 +620,50 @@ int uct_iface_is_reachable(uct_iface_h iface, const struct sockaddr *addr);
 
 /**
  * @ingroup UCT_RESOURCE
+ * @brief Open a completion file descriptor for polling.
+ *
+ * @param [in] iface       Handle to an open communication interface.
+ *
+ * @return valid OS file-descriptor if successful, -1 otherwise.
+ */
+UCT_INLINE_API int uct_iface_open_fd(uct_iface_h iface)
+{
+    return iface->ops.iface_open_fd(iface);
+}
+
+
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Relaxes a signaled completion file descriptor.
+ *
+ * @param [in] iface       Handle to an open communication interface.
+ * @param [in] comp_fd     Completion file descriptor.
+ *
+ * @return Error code.
+ */
+UCT_INLINE_API ucs_status_t uct_iface_drain_fd(uct_iface_h iface, int comp_fd)
+{
+    return iface->ops.iface_drain_fd(iface, comp_fd);
+}
+
+
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Closes a completion file descriptor.
+ *
+ * @param [in] iface       Handle to an open communication interface.
+ * @param [in] comp_fd     Completion file descriptor.
+ *
+ * @return Error code.
+ */
+UCT_INLINE_API void uct_iface_close_fd(uct_iface_h iface, int comp_fd)
+{
+    return iface->ops.iface_close_fd(iface, comp_fd);
+}
+
+
+/**
+ * @ingroup UCT_RESOURCE
```

Comment:
Is this complete API ? What does the drain do ? What kinda events are signaled ? It doesn't look as a complete API